### PR TITLE
[clojure/de] Translate clojure to german

### PR DESCRIPTION
**Pull request by denniskeller**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: de-de/clojure-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+@counter ; => 5
+
+; Andere STM Konstrukte sind refs und agents.
+; Refs: http://clojure.org/refs
+; Agents: http://clojure.org/agents
+```
+
+### Weiterführende Literatur
+
+Das ist alles andere als erschöpfend, aber hoffentlich ist es genug, um dich auf die Beine zu stellen.
```
Comment (mxschumacher) [inline de-de/clojure-de.html.markdown]: Unfortunately, this is too literal for a translation, both "erschöpfend" and "dich auf die Beine stellen" don't quite work here

File: de-de/clojure-de.html.markdown [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+; Maps
+;;;;;;;;;;
+
+; Hash maps und Array maps teilen sich ein Interface. Hash maps haben eine schnellere Zugriffszeit,
+; aber behalten keine Schlüsselreihenfolge.
+(class {:a 1 :b 2 :c 3}) ; => clojure.lang.PersistentArrayMap
+(class (hash-map :a 1 :b 2 :c 3)) ; => clojure.lang.PersistentHashMap
+
+; Arraymaps werden durch die meisten Operatioen automatisch zu Hashmaps,
+; wenn Sie groß genug werden. Das heißt du musst dich nicht darum sorgen.
```
Comment (mxschumacher) [inline de-de/clojure-de.html.markdown]: "sie" is not capitalised here

wenn => sobald

File: de-de/clojure-de.html.markdown [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+language: clojure
+filename: learnclojure-de.clj
+contributors:
+    - ["Adam Bard", "http://adambard.com/"]
+translators:
+    - ["Dennis Keller", "https://github.com/denniskeller"]
+lang: de-de
+---
+
+Clojure ist ein Lispdialekt, der für die Java Virtual Maschine entwickelt worden ist. Sie hat eine stärkere Betonung auf pure [funktionale Programmierung](https://en.wikipedia.org/wiki/Functional_programming) als Common Lisp. Jedoch besitzt sie je nach Bedarf mehrere [STM](https://en.wikipedia.org/wiki/Software_transactional_memory) Werkzeuge zur Handhabung von Zustand.
```
Comment (mxschumacher) [inline de-de/clojure-de.html.markdown]: I'd recommend "reine funktionale Programmierung" intead of "pure"

File: de-de/clojure-de.html.markdown (hunk 1 above)
Comment (verhovsky) [reply de-de/clojure-de.html.markdown:410]: If you come up with an alternative, please submit a PR.

**Pull request merged**